### DISCUSSION TITLE
How should I make it so that player has to go through every choice before proceeding?

### DISCUSSION THREAD

**Discussion by memory942** [dialogic-godot/dialogic#2274] [Q&A]
If there are 4 dialogue choices from "A"~"D", unless all of them are seen by the player, I want the game to keep returning to this choice choosing screen.
And only when the player has seen all 4 choices, proceed to the next part of the dialogue.
How should I achieve this?

**Comment by run-666-art** [comment]
you can create a Label on the timeline where the player need to pick the choices and at the end of every choice you use the Jump option to return to the Label. at the end of the choices, you do whatever you want, like close the dialogue.
